<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QWidget" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1100</width>
     <height>750</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>222</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>音乐播放列表</string>
    </property>
    <widget class="QListWidget" name="music_list_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>202</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>202</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="music_add_3">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="music_delete_3">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>视频播放</string>
    </property>
    <widget class="QVideoWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>46</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_search_3"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search_3">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>521</width>
      <height>181</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00/3:21</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>501</y>
      <width>849</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>721</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>552</y>
      <width>831</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btn_pre_3">
       <property name="text">
        <string>上一首</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_play_3">
       <property name="text">
        <string>播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_next_3">
       <property name="text">
        <string>下一首</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>播放模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="play_model_3">
       <item>
        <property name="text">
         <string>列表播放</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>随机播放</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>循环播放</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>音量控制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volume_control_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="lyric_widget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>551</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget::item:selected 
{
	background-color: rgba(231, 124, 142,100);
	color: rgb(0, 0, 0);
	font-size:20px;
}
QListWidget::item
{
	color: rgba(47, 47, 53, 150);
	font-size: 20px;
}</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_return">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_disconnect">
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IP：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="host_ip"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>端口号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_ports"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
